archiver.go

```diff
@@ -1440,17 +1440,17 @@ func DeleteArchivedRuns(ctx context.Context, config *Config, db *sqlx.DB, s3Clie
 	defer rows.Close()
 
 	var runID int64
-	var isActive bool
+	var exitType string
 	runCount := 0
 	runIDs := make([]int64, 0, archive.RecordCount)
 	for rows.Next() {
-		err = rows.Scan(&runID, &isActive)
+		err = rows.Scan(&runID, &exitType)
 		if err != nil {
 			return err
 		}
 
 		// if this run is still active, something has gone wrong, throw an error
-		if isActive {
+		if exitType != "C" && exitType != "E" && exitType != "I" {
```

Comment:
Shouldn't it just be whether exit type is not null? Is the unit test updated? Wouldn't a null value for exit type blow up here? (need to use either `*string` or `null.String` for it to deal with `NULL` correctly)